src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -155,8 +174,8 @@ public bool IsInitialized
         {
             await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
Can we remove this?

Reply:
we have DTE call in this method later, we can't remove this